kernel/base/src/main/java/com/twosigma/beakerx/table/TableDisplay.java

```diff
@@ -627,7 +629,17 @@ public void setDoubleClickAction(Object listener) {
   }
 
   private void handleDoubleClick(Message message) {
-    handleCommEventSync(message, CommActions.DOUBLE_CLICK, this::onDoubleClickAction);
+    if (isCorrectEvent(message, CommActions.DOUBLE_CLICK)) {
+      handleCommEventSync(message, CommActions.DOUBLE_CLICK, this::onDoubleClickAction);
+    }
+  }
+
+  private boolean isCorrectEvent(Message message, CommActions commActions) {
```

Comment:
unclear to me what this function is for?
when does it return false? wouldn't this be an error condition?